juju/juju
# Update dummy-storage Charm based on the review of persistent storage requirement.

## Description of change
Update dummy-storage Charm to support block device storage.
Update the range of storage slot to make all of them optional by default - easy for testing.

## QA steps
1. set credentials, bootstrap a controller on AWS.
2. $ export JUJU_REPOSITORY=<path_to>/juju/acceptancetests/repository
3. $ juju deploy ${JUJU_REPOSITORY}/charms/dummy-storage --storage single-blk=ebs
4. wait until the Charm deployment is finished.
5. $ juju storage, the output should look like this:
[Storage]
Unit             Id            Type   Pool  Provider id            Size    Status    Message
dummy-storage/4  single-blk/8  block  ebs   vol-07f5261398bc0e268  1.0GiB  attached

## Documentation changes
N/A

## Bug reference
N/A
